src/OrleansRuntime/Streams/PersistentStream/PersistentStreamPullingManager.cs

```diff
@@ -248,7 +244,7 @@ public Task QueueDistributionChangeNotification()
                 // Just ignore this exception and proceed as if Initialize has succeeded.
                 // We already logged individual exceptions for individual calls to Initialize. No need to log again.
             }
-            logger.Info((int)ErrorCode.PersistentStreamPullingManager_09, "Took {0} new queues under my responsibility: {1}", agents.Count,
+            logger.Info((int)ErrorCode.PersistentStreamPullingManager_08, "Took {0} new queues under my responsibility: {1}", agents.Count,
```

Comment:
I wanted to order them sequentially, to make sure we use each number only once. Otherwise, it is hard to check that.